python/prophet/models.py

```diff
@@ -54,7 +54,7 @@ def sampling(self, stan_init, stan_data, samples, **kwargs) -> dict:
 
 
 class CmdStanPyBackend(IStanBackend):
-    CMDSTAN_VERSION = "2.31.0"
+    CMDSTAN_VERSION = "2.33.0"
```

Comment:
latest is 2.33.1, https://github.com/stan-dev/cmdstan/releases

Reply:
updated and tested on my M2